pyproject.toml

```diff
@@ -47,6 +47,7 @@ pg8000 = {version = "^1.30.3", optional = true}
 torch = {version = ">=2.0.0, !=2.0.1, !=2.1.0", optional = true}
 websockets = "^12.0"
 docstring-parser = "^0.15"
+lancedb = "^0.3.3"
```

Comment:
Could you please add this an an option dependency, and make an extra called lancedb? e.g. you you can do `pip install pymemgpt[lancedb]` -- we are trying to keep the dependencies are minimal as possible

Reply:
Could you please change this to `lancedb = {version = "^0.3.3", optional = true}`?